Modules/Module_Import/Import/xASL_imp_DetermineStructureFromSourcedata.m

```diff
@@ -155,20 +175,20 @@ function xASL_imp_PrintRun(x,thisSubject,thisVisit,thisRun)
 
 end
 
-
-
+% -----------------------------------------------------------------
 %% Add subjects to overview
+% -----------------------------------------------------------------
 function x = xASL_imp_AddSubjectOverview(x)
     
     for iSubject=1:numel(x.modules.import.listsIDs.subjectIDs)
         thisSubject = x.modules.import.listsIDs.subjectIDs{iSubject};
         x = xASL_imp_AddSubject(x, thisSubject, iSubject);
     end
-
 end
 
-
+% -----------------------------------------------------------------
```

Comment:
I would add a bit more space between functions, the way you did it is also fine though.